keyboards/nifty_numpad/readme.md

```diff
@@ -0,0 +1,35 @@
+# Nifty Numpad
+
+![Nifty Numpad](https://gitlab.com/Acliad/nifty-numpad/-/raw/main/Docs/Render%20(Blue,%20SS%20screws).png)
```

Comment:
Image not found? 

Also, please rehost on Imgur or similar hosting site.

Reply:
That's odd, I tested it and that repo isn't private. Regardless, I've rehosted it on Imgur and updated the readme.